tests/test_sqlPreFlight.py

```diff
@@ -0,0 +1,240 @@
+# This file is part of daf_butler.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (http://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import os
+import unittest
+
+import lsst.utils.tests
+
+from lsst.daf.butler.core.butlerConfig import ButlerConfig
+from lsst.daf.butler.core.datasets import DatasetType
+from lsst.daf.butler.core.registry import Registry
+from lsst.daf.butler.core.storageClass import StorageClass
+
+"""Tests for SqlRegistry.
+"""
```

Comment:
This docstring seems superfluous.  I'm also not sure it works as a docstring anyway if it's below all of the imports.